admin-front/src/components/TimeSeries/functions/getDateFormat.ts

```diff
@@ -0,0 +1,28 @@
+import { Month, Resolution, ResolutionValues } from '../utility';
+
+interface GetDateFormatParams {
+  resolution: Resolution;
+  month: Month;
+}
+
+export const getDateFormat = ({
+  resolution,
+  month,
+}: GetDateFormatParams): string => {
+  let format = 'DD/MM';
+  const individualMonthIsSelected = month !== 'All';
+
+  if (resolution === ResolutionValues.HOUR) {
+    format = 'DD/MM HH:MM';
+  }
+
+  if (resolution === ResolutionValues.DAY && individualMonthIsSelected) {
+    format = 'DD';
+  }
+
+  if (resolution === ResolutionValues.MONTH) {
```

Comment:
I think here a `switch` would be better readable? Because all of these if statements are about the same variable (`resolution`).